test/signing_suite.go

```diff
@@ -0,0 +1,56 @@
+package test
+
+import (
+	"testing"
+
+	"github.com/filecoin-project/go-f3/f3"
+	"github.com/stretchr/testify/require"
+)
+
+type SigningSuite struct {
+	signer   f3.Signer
+	verifier f3.Verifier
+}
+
+func NewSigningSuite(signer f3.Signer, verifier f3.Verifier) *SigningSuite {
+	return &SigningSuite{
+		signer:   signer,
+		verifier: verifier,
+	}
+}
+
+func (s *SigningSuite) TestSignAndVerify(t *testing.T) {
+	pubKey := s.signer.GenerateKey()
+	msg := []byte("test message")
+	sig, err := s.signer.Sign(pubKey, msg)
+	require.NoError(t, err)
+
+	err = s.verifier.Verify(pubKey, msg, sig)
```

Comment:
Test also that Verify isn't just `return nil` by checking an invalid signature. Ditto for aggregation